### ISSUE TITLE
Why is karma pinned to 5.2.0 in @angular-devkit/build-angular?

### ISSUE THREAD

**Issue by qortex** [angular/angular-cli#19945]
# 🐞 Bug report

### Command (mark with an `x`)

- [ ] new
- [ ] build
- [ ] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [x] update
- [ ] lint
- [ ] extract-i18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc

### Description

When running `ng update karma` (I'm currently on 5.2.0) I get:
```
Fetching dependency metadata from registry...
                  Package "@angular-devkit/build-angular" has an incompatible peer dependency to "karma" (requires "^5.2.0", would install "6.1.0").
```

I use `yarn`.

From [this file](https://github.com/angular/angular-cli/blob/master/packages/angular_devkit/build_angular/package.json), I understand version 6 should be permitted. Why is it not the case?

## 🌍 Your Environment
<pre><code>
   _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/
    

Angular CLI: 11.1.2
Node: 14.4.0
OS: linux x64

Angular: 11.1.1
... animations, cdk, common, compiler, compiler-cli, core, forms
... language-service, localize, material, platform-browser
... platform-browser-dynamic, router
Ivy Workspace: Yes

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1101.2
@angular-devkit/build-angular   0.1101.2
@angular-devkit/core            11.1.2
@angular-devkit/schematics      11.1.2
@angular/cli                    11.1.2
@angular/flex-layout            11.0.0-beta.33
@schematics/angular             11.1.2
@schematics/update              0.1101.2
rxjs                            6.6.3
typescript                      4.1.3
</code></pre>

**Comment by alan-agius4** [comment]
Support for Karma version 6 will be available in Angular CLI version 11.2.0.

You can try the pre-release using `ng update @angular/cli --next`

**Comment by qortex** [comment]
Ah right, sorry, misread the changelogs & pre-release.
